packages/wonder-blocks-core/components/media-layout.js

```diff
@@ -29,7 +29,16 @@ type Props = {
 
     /** The contents of the page */
     children: React.Node,
-};
+
+    /**
+     * Style the layout container
+     */
+    style?: any,
+
+    // TODO(kevinb): provide a way to "mixin" allowed roles and aria props
+    role?: "dialog",
+    "aria-labelledby"?: string,
```

Comment:
We shouldn't be adding aria props like these, but this will have to do for now.

Reply:
The moment we have support for React 16 we can rewrite MediaLayout to just be a passthrough component so this won't be needed anymore!